enhancements/network/user-defined-network-segmentation.md

```diff
@@ -836,6 +848,11 @@ Having the prefix avoids conflicting with existing NADs who has the same `metada
 For example:
 Given the CR meta.name is `db-network`,the NAD metadata.name will be `cluster.udn.db-network`.
 
+If a VRF name is provided in the cluster-scope CRD instance, it will be set to an homonimous field in the NAD NetConf. If the 
```

Comment:
The VRF field eventually translated to some ID and assigned to corresponding  field in the NetConf (NAD spec.config) that have less restrictions on length.
What does the generated ID (e.g.: udn-mynet-vrf) represent? which entity consumes it?

Can the NetConf.Name be used instead?
It has some similar characteristics.

Reply:
The meaning and use of the VRF is not introduced in this proposal amendment, it was already introduced and explained on the original enhancement. 

The generation of VRF names was assumed as an implementation detail and not specified in the enhancement.
This proposal just introduces a 

This is just optional user provided name for it and surfacing the name generation that was already assumed.